staging/src/k8s.io/legacy-cloud-providers/azure/auth/azure_auth.go

```diff
@@ -132,6 +142,78 @@ func GetServicePrincipalToken(config *AzureAuthConfig, env *azure.Environment) (
 	return nil, ErrorNoAuth
 }
 
+// GetMultiTenantServicePrincipalToken creates a new multi-tenant service principal token based on the configuration
+// PrimaryToken of the result is for the AAD Tenant specified by TenantID, and AuxiliaryToken of the result is for the AAD Tenant specified by CrossTenantNetworkResourceConfig.TenantID
+func GetMultiTenantServicePrincipalToken(config *AzureAuthConfig, env *azure.Environment) (*adal.MultiTenantServicePrincipalToken, error) {
```

Comment:
is it possible for write a common func for `GetMultiTenantServicePrincipalToken`, `GetNetworkResourceServicePrincipalToken`, the func logic is quite similar

Reply:
Thank you. Updated.